src/Raven.Server/Web/System/TcpConnectionInfoHandler.cs

```diff
@@ -70,22 +72,25 @@ public Task GetRemoteTaskTopology()
         }
 
         [RavenAction("/info/remote-task/tcp", "GET", AuthorizationStatus.RestrictedAccess)]
-        public Task GetRemoteTaskTcp()
+        public async Task GetRemoteTaskTcp()
         {
             var remoteTask = GetStringQueryString("remote-task");
             var database = GetStringQueryString("database");
+            var verifyDatabase = GetBoolValueQueryString("verify-database", false);
+            if (verifyDatabase.HasValue && verifyDatabase.Value)
```

Comment:
Why do we have this? Worst case, we'll get an error on the TCP connection, no?

Reply:
There is a failing subscription test where we disabled the database and the get tcp command is not failing and expected to fail.
The subscription code assumes that the get tcp info will fail if the database is disabled.
I have fixed it so if the active connection could deal with disabling a database but @maximburyak  reviewed it and thought it is the wrong fix and that the tcp info call should fail if the database is disabled and this is why i added this.